datameta/views/admin.py

```diff
@@ -50,11 +50,11 @@ def v_admin_put_request(request):
     # Parse request
     accept = False
     try:
-        reg_req_id             = int(request.json_body['id'])
+        reg_req_id             = str(request.json_body['id'])
```

Comment:
`json_body` is a string dict
```suggestion
        reg_req_id             = request.json_body['id']
```